### PR TITLE
Added script for coverage Policies_Security

### PR DESCRIPTION
**Pull request by Aleksandriys**
@aarotar 
Please review.

### CODE REVIEW

**Comment by Aleksandriys** [comment]
Plase review new script

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 1]
```diff
@@ -0,0 +1,86 @@
+ -----  Name of requirement that is covered-----
+ ----- Certificate have EMPTY in pt_preloaded JSON of module_config section
+ ----- Verification criteria------
+ -----  SDL must continue working as assigned.
+
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, please you in script the format of header from our approved script template.

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+--[[ Required Shared libraries ]]  
+local mobile_session = require('mobile_session')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+NewTestSuiteNumber = 0
+require('user_modules/AppTypes')
+
+--[[ Local preparing ]]
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys , where block --[[ Local preparing ]] is from?? It should be renamed to --[[ Local functions ]] or --[[ Preconditions ]] - depending on its purpose. We should use block names from our template.

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]  
+local mobile_session = require('mobile_session')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys , commonPreconditions are not used in script - no need to declare them.

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+NewTestSuiteNumber = 0
+require('user_modules/AppTypes')
+
+--[[ Local preparing ]]
+commonSteps:DeleteLogsFileAndPolicyTable()
+Test = require('connecttest')
+
+
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys,  please avoid redundant empty lines in script

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+Test["ForceKill"] = function (self)
+os.execute("ps aux | grep smart | awk \'{print $2}\' | xargs kill -9")
+os.execute("sleep 1")
+
+return Test
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, there is no need in code **return Test**  in script

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+--[[ Test Case]]
+commonFunctions:newTestCasesGroup("Test Case")
+function Test:Initiate_PTU_with_Certificate_Empty_Value()
+  policyUpdate(self, "/tmp/fs/mp/images/ivsu_cache/ptu.json")
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+Test["ForceKill"] = function (self)
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys use for force SDL stop appropriate common function.

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 7]
```diff
@@ -0,0 +1,86 @@
+ -----  Name of requirement that is covered-----
+ ----- Certificate have EMPTY in pt_preloaded JSON of module_config section
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, check syntax of requirement summary

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+        pathToSnaphot
+      )
+   EXPECT_HMICALL("BasicCommunication.SystemRequest")
+     :Do(function(_,data)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end)
+      EXPECT_RESPONSE(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+      :Do(function(_,data)
+          --Copy of JSON file in /tmp/fs/mp/images/ivsu_cache/
+          --Example:os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/ptu.json /tmp/fs/mp/images/ivsu_cache/")
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys , remove commented code from script

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+local function policyUpdate(self)
+  local pathToSnaphot = nil
+  EXPECT_HMICALL ("BasicCommunication.PolicyUpdate")
+  :Do(function(_,data)
+      pathToSnaphot = data.params.file
+      self.hmiConnection:SendResponse(data.id, "BasicCommunication.PolicyUpdate", "SUCCESS", {})
+    end)
+   local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
+   EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {url = "http://policies.telematics.ford.com/api/policies"}}})
+    :Do(function(_,data)
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, avoid unused variables (data variable isn't used in function)

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+          requestType = "PROPRIETARY",
+          url = "http://policies.telematics.ford.com/api/policies",
+          appID = self.applications ["Test Application"],
+          fileName = "sdl_snapshot.json"
+        },
+        pathToSnaphot
+      )
+    end)
+  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY" })
+  :Do(function(_,data)
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, avoid unused variables (data variable isn't used in function)

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+          fileName = "sdl_snapshot.json"
+        },
+        pathToSnaphot
+      )
+   EXPECT_HMICALL("BasicCommunication.SystemRequest")
+     :Do(function(_,data)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end)
+      EXPECT_RESPONSE(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+      :Do(function(_,data)
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, avoid unused variables (data variable isn't used in function)

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+          --Copy of JSON file in /tmp/fs/mp/images/ivsu_cache/
+          --Example:os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/ptu.json /tmp/fs/mp/images/ivsu_cache/")
+          self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+            {
+              policyfile = "/tmp/fs/mp/images/ivsu_cache/ptu.json"
+            })
+        end)
+      :Do(function(_,data)
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys, avoid unused variables (data variable isn't used in function)

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+          EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UP_TO_DATE"})
+        end)
+    end)
+end
+--[[ Test Case]]
+commonFunctions:newTestCasesGroup("Test Case")
+function Test:Initiate_PTU_with_Certificate_Empty_Value()
```
Comment (aarotar) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys , it is unclear, why for verifying requirement, PTU is used. Requirement is related to SDL first startup with specific preloaded. So, requirement is not checked in script at all!

**Comment by Aleksandriys** [comment]
@istoimenova and @maria-mih please review.

**Pull request merged**